pool/pooler.go

```diff
@@ -155,3 +161,9 @@ type Pooler interface {
 	ExecuteAsync(expr string, args interface{},
 		mode Mode) *tarantool.Future
 }
+
+// PoolerTopology is the interface that must be implemented by a connection pool and describe topology methods.
+type PoolerTopology interface {
+	Add(ctx context.Context, instance Instance) error
+	Remove(name string) error
+}
```

Comment:
nit: move this interface so it would be before the `Pooler`. Something like this:
```go
// ...
type PoolerTopology interface {
     ...
}

// ...
type Pooler interface {
    ...
}
```

This change is not crucial and could be ignored, up to you.